### PR TITLE
Macos new builders

### PR DESCRIPTION
**Pull request by DavHau**
based on https://github.com/holochain/holochain-infra/pull/26

- Adds two new builder macos-03, macos-04
- add updated installation instructions

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -34,10 +34,6 @@
       url = "https://github.com/neonphog.keys";
       flake = false;
     };
-    keys_thedavidmeister = {
```
Comment (steveej) [inline flake.nix]: why remove this key? @thedavidmeister is actively working with us

File: modules/nixos/nix-build-distributor.nix [hunk 2]
```diff
@@ -25,6 +27,26 @@
...
+      maxJobs = 8;
+      supportedFeatures = config.nix.settings.experimental-features;
+    }
+
+    # macos-04
+    {
+      hostName = "167.235.38.111";
+      sshUser = "builder";
+      protocol = "ssh-ng";
+      system = "aarch64-darwin";
```
Comment (steveej) [inline modules/nixos/nix-build-distributor.nix]:
```suggestion
      system = "x86_64-darwin";
```

i'd like to balance capacity and improve resilience for `x86_64-darwin`

File: modules/nixos/nix-build-distributor.nix [hunk 3]
```diff
@@ -25,6 +27,26 @@
       speedFactor = 10;
```
Comment (steveej) [inline modules/nixos/nix-build-distributor.nix]:
```suggestion
```

**Review by steveej — CHANGES_REQUESTED** [review]
nice, thanks a lot! a few changes i'd like to make before we make the final deployment and merge.

File: flake.nix (hunk 1 above)
Comment (thedavidmeister) [reply flake.nix]: indeed

Comment (DavHau) [reply flake.nix]: I just removed this because the keys from github changed and I wasn't sure if this is supposed to be.

Comment (thedavidmeister) [reply flake.nix]: the keys changed? as in at my end?

Comment (DavHau) [reply flake.nix]: Your keys on github are not the same as a few weeks ago. That's all I know.

Comment (thedavidmeister) [reply flake.nix]: oh, i added a new one for a new device yeah

Comment (DavHau) [reply flake.nix]: Alright, thanks for clarifying. I'll add your keys back tomorrow.

Comment (DavHau) [reply flake.nix]: Added back the keys and updated the lock file

**Review by steveej — APPROVED** [review]

**Pull request merged**